app/Http/Controllers/Api/V1/LtiTool/LtiToolSettingsController.php

```diff
@@ -55,12 +61,19 @@ public function index(Request $request, Organization $suborganization)
 
     /**
      * Get Lti Tool Setting
+     * 
      * Get the specified lti tool setting data.
+     * 
      * @urlParam id required The Id of a lti_tool_settings table Example: 1
-     * @urlParam Organization $suborganization
+     * @urlParam Organization $suborganization The Id of suborganization Example: 1
```

Comment:
Does not seem to be valid format for suborganization in in LtiToolSettingsController.php.